app/views/offers/index/_single_search_result.html.slim

```diff
@@ -9,6 +9,4 @@ li.JS-trigger-marker.Listing-results__listitem id="result-offer-#{offer.id}"
     span.Listing-results__emphasis.Listing-results__organization
       = offer.organization_display_name
     - if offer.location
-      = offer.location_address
-    - else
-      = t 'offers.index.location_everywhere'
```

Comment:
Bitte auch Übersetzung in der locale löschen (falls nicht anderswo benutzt, was ich nicht glaube). Sonst gibt es verwaiste Übersetzungen.